### DISCUSSION TITLE
Welcome new committer: shihuili1218

### DISCUSSION THREAD

**Discussion by fengjiachun** [sofastack/sofa-jraft#1143] [Announcements]
Hi Community,

I am pleased to announce that @shihuili1218 has been invited to become a SOFAJRaft committer.

Please join me in congratulating him!

![image](https://github.com/user-attachments/assets/9269ebd7-b6a3-4df9-9a43-ca5f5000990e)

**Comment by shihuili1218** [comment]
❤️‍🔥❤️‍🔥❤️‍🔥❤️‍🔥
